common/djangoapps/student/models.py

```diff
@@ -1442,6 +1442,28 @@ def enrollments_for_user(cls, user):
         return cls.objects.filter(user=user, is_active=1)
 
     @classmethod
+    def preloaded_enrollments_for_user(cls, user):
+        """
```

Comment:
Sounds good to me, leave it as-is.  I wonder if clarifying that `enrollments_for_user` returns a QuerySet is useful, but maybe that's implicit.